### ISSUE TITLE
Action container is not visible in uui-ref-node when focusing on child buttons

### ISSUE THREAD

**Issue by MMasey** [umbraco/Umbraco.UI#947]
### Which *exact* UI Library version are you using? For example: 1.0.1 - don't just write v1.

1.11.0

### Bug summary

When using the keyboard to navigate the Node Ui element, focusing on an action item, such as a button, doesn't make it visible

### Steps to reproduce

1. Run story book
2. Navigate to Displays -> Node -> Docs
3. Click on the element and then hit tab on your heyboard to try and pake the action button appear.

### Expected result / actual result

**Expected results**
![image](https://github.com/user-attachments/assets/18ffdf7f-d377-4d07-b439-934f641e2bb1)

**Actual results**
![image](https://github.com/user-attachments/assets/f5aaed55-284d-496c-a1f2-fff99d2a2c1a)

**Comment by lattaai3** [comment]
Hello, I tried to fix the issue

This is what I did:

Updated the UUIRefNodeElement to make the action button visible when focused using keyboard navigation. Added CSS styles to show the actions container on focus and hover.

I also created pull request: https://github.com/umbraco/Umbraco.UI/pull/948


> [!CAUTION]
> Disclaimer: This fix was created by Latta AI and you should never merge before you check the correctness of generated code!
>
> The fix provided by Latta AI might not be complete and it can serve as an inspiration.

---
This bug was fixed for free by Latta AI - https://latta.ai/ourmission

If you no longer want Latta AI to attempt fixing issues on your repository, you can block this account.

**Comment by MMasey** [comment]
This is a duplicate of https://github.com/umbraco/Umbraco.UI/pull/948
